### PR TITLE
Create proverb list 

### PR DESCRIPTION
**Pull request by salih18**
Created proverb list component . For now it uses the dummy data. 

Created FlexTable component to render proverblist. 

Created Home page .

### CODE REVIEW

File: client/src/components/FlexTable.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import "./FlexTable.css";
+import Icon from "./Icon";
+
+const FlexTable = ({ titleData = [], data = [], iconClick, tableId }) => {
+  return (
```
Comment (frankPairs) [inline client/src/components/FlexTable.js:6]: I would say to split this component a bit. It is a bit difficult to read. Maybe you could create different smaller components like TableHeader, or TableCell

File: client/src/components/FlexTable.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+              className={`flexTable-cell ${item.fieldName}-cell column-heading`}
+            >
+              {" "}
+              {item.title}
+            </div>
+          ))}
+        </div>
+      </div>
+      {data.map((item, i) => (
+        <React.Fragment key={"row" + i}>
```
Comment (frankPairs) [inline client/src/components/FlexTable.js:25]: I think this React.Fragment is not necessary.

**Review by frankPairs — APPROVED** [review]
good job Salih! I only put a comment related to the FlexTable component. the rest looks perfect

File: client/src/components/FlexTable.js (hunk 1 above)
Comment (salih18) [reply client/src/components/FlexTable.js:6]: I would also say the same. It is better to divide into as your suggestion. Thanks.

File: client/src/components/FlexTable.js (hunk 2 above)
Comment (salih18) [reply client/src/components/FlexTable.js]: Yes, certainly.

**Pull request merged**